staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4710,8 +4712,47 @@ type ResourceQuotaSpec struct {
 	// If not specified, the quota matches all objects.
 	// +optional
 	Scopes []ResourceQuotaScope `json:"scopes,omitempty" protobuf:"bytes,2,rep,name=scopes,casttype=ResourceQuotaScope"`
+	// ScopeSelector is also a collection of filters like Scopes that must match each object tracked by a quota
```

Comment:
This should be lower cased (publicly accessible API docs should use the field names an end user would see).  `scopeSelector ... scopes ...`.

As a follow up, I'd like to see a much better godoc here that describes how a user would use this.  Can be done after this merges.

Reply:
You need to define whether this is `OR` with Scopes or `AND` with Scopes.